static/js/warsztatywww.js

```diff
@@ -111,7 +111,7 @@ $(function () {
     $('[data-toggle="popover"]').popover();
 
     // Automatically hide 'Saved successfully' alerts after 4 seconds
-    $(".alert").delay(4000).fadeTo(500, 0).slideUp(500, function(){
+    $(".alert-info, .alert-success").delay(4000).fadeTo(500, 0).slideUp(500, function(){
```

Comment:
Hard-coded pixel coordinates seems like a bad idea. Get the viewport size and divide by 2.

Reply:
I don't see any pixels here, only time in ms :P

And stop complaining about things that were not changed in pull request reviews, that's not what they are for - open an issue or fix it yourself in another PR